bigtable/client/table.h

```diff
@@ -41,6 +41,62 @@ inline std::string TableName(std::shared_ptr<DataClient> client,
                              std::string const& table_id) {
   return InstanceName(std::move(client)) + "/tables/" + table_id;
 }
+/**
+ * No Exception namespace contains the public version of interface without
```

Comment:
Consider:

```C++
/**
  * This namespace contains implementations of the API that do not raise exceptions.
  * It is subject to change without notice, and therefore, not recommended for direct
  * use by applications.
  */
```

I think we should refactor the code in `bigtable::noex::` to `bigtable/client/internal/`.

Reply:
agree, I will move to bigtable/client/internal